docs/fundamentals/jcloud/autoscale.md

```diff
@@ -0,0 +1,54 @@
+# Autoscaling
+
+JCloud is Kubernetes based and one of the biggest benefits of Kubernetes is the ability to autoscale your applications
+on-demand. In JCloud, autoscaling function is naturally offered as well, so you can maintain the [serverless](https://en.wikipedia.org/wiki/Serverless_computing) deployments in a cost-effective way with no headache of setting the [right number of replicas](https://docs.jina.ai/how-to/scale-out/#scale-out-your-executor) anymore!
+
+### Configurations
```

Comment:
Is there a reason this is a third level header (`###`) instead of second level (`##`)?
```suggestion
## Configurations
```

Reply:
We originally thought ### header's size would be more appropriate but I just did comparison, looks like they are the same. Fixing it then!